locales/pl.json

```diff
@@ -0,0 +1,74 @@
+{
+	"select-user-type.who_you_are": "Kim jesteś? (będziesz to mógł zmienić później)",
+	"select-user-type.driver": "Kierowca",
+	"select-user-type.passenger": "Pasażer",
+	"request-phone.type_your_phone": "Proszę przesłać do nas swój numer telefonu. Nie udostępnimy go bez twojej wyraźnej zgody.",
+	"request-phone.button_text": "Wyślij numer",
+	"request-phone.all_set": "👌 Wszystko jest już ustawione, możesz już zamówić przejazd!",
```

Comment:
The "już" repetition is a stylistic error. My suggestion would be "Wszystko jest już ustawione, możesz zamówić przejazd!"